src/components/VideoChatMenu.js

```diff
@@ -0,0 +1,69 @@
+import React, {Component} from 'react';
+import Popover from "./Popover";
+import PropTypes from 'prop-types';
+import withWindowDimensions, {windowDimensionsPropTypes} from "./withWindowDimensions";
+import MenuItem from './MenuItem';
+import openURLInNewTab from '../libs/openURLInNewTab';
+import ZoomIcon from '../../assets/images/Zoom - Blue.svg';
+import GoogleMeetIcon from '../../assets/images/google-meet.svg'
+import CONST from '../CONST';
+
+const propTypes = {
+    // State that determines whether to display the create menu or not
+    isVisible: PropTypes.bool.isRequired,
+
+    // 
+    onClose: PropTypes.func.isRequired,
+
+    ...windowDimensionsPropTypes,
+};
+
+class VideoChatMenu extends Component {
+    constructor(props) {
+        super(props)
+    }
+
+    render() {
+        const menuItemData = [
+            {
+                icon: ZoomIcon,
+                text: 'Zoom',
+                onPress: () => openURLInNewTab(CONST.NEW_ZOOM_MEETING_URL),
+            },
+            {
+                icon: GoogleMeetIcon,
+                text: 'Google Meet',
+                onPress: () => openURLInNewTab(CONST.NEW_GOOGLE_MEET_MEETING_URL),
+            },
+        ].map(item => ({
+            ...item,
+            onPress: () => {
+                this.props.onItemSelected();
+                item.onPress();
+            },
+        }));
+        return (
+            <Popover
+                onClose={this.props.onClose}
+                isVisible={this.props.isVisible}
+                anchorPosition={{
+                    left: this.props.windowWidth - 250,
+                    top: 50
+                }}
```

Comment:
Currently, the approach is to have the menu display relative to the top right corner of the window (because the video chat UI element does as well). 
    Pro: very simple approach
    Con:  not _actually_ tied to the button

My alternative approach would be to have the UI element (the 📞 button) give its coordinates back to the container (`ReportScreen`) when clicked, and then pass those coordinates to this component to set its anchor position. 
    Pro: location is relative to button,
    Con: feels over-engineered for the task at hand

For a second opinion I asked @roryabraham which approach he likes more, and it turned out to be the latter. Now it's a 1v1 tie, so I'm gonna ask for a third opinion. 😄  @tgolen, would you mind weighing in for the tie breaker? 

For context, here is Shawn's mock-up:
![image](https://user-images.githubusercontent.com/2319350/111755245-32e25680-8899-11eb-9e4d-7c6b43485b63.png)

Reply:
I like the second option too. Being more explicit with the coordinates will make it easier to debug later I feel. I usually shudder at absolutely positioning things.

Reply:
Okie doke! I appreciate the input. It will make our jobs easier in the future should we ever choose to move the button. 👍

Reply:
Sounds like that solution might work well for when the window resizes too? Or in the very least, keep that situation in mind as you test this.